### PR TITLE
Add jBPM Console

### PR DESCRIPTION
**Pull request by yakimant**
https://github.com/status-im/infra-logos-office/issues/32

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -2,7 +2,8 @@
 - import_tasks: auth.yml
 - import_tasks: query.yml
 - import_tasks: config.yml
+- import_tasks: jbpm-console.yml
 - import_tasks: script.yml
 - import_tasks: docker.yml
-- import_tasks: users.yml
+- import_tasks: db.yml
```
Comment (jakubgs) [inline tasks/main.yml]: Maybe `access.yml` would be more descriptive?

File: tasks/db.yml [hunk 2]
```diff
@@ -12,7 +13,48 @@
...
+    login_password: '{{ openkm_db_pass | mandatory }}'
+    login_host:     'localhost'
+    query: |
+      UPDATE okm_config
+      SET cfg_value = %s
+      WHERE cfg_key = 'workflow.adapter.pasword'
+    positional_args:
+      - '{{ openkm_jbpm_admin_password | mandatory }}'
+
+# TODO: jbpm doesn't like it, maybe due to bcrypt version 2a vs 2b
```
Comment (jakubgs) [inline tasks/db.yml]: What does "doesn't like it" mean? You can't log in or does it explode? Is there an error?

**Review by jakubgs — APPROVED** [review]

File: tasks/db.yml (hunk 2 above)
Comment (yakimant) [reply tasks/db.yml]: You can't login with such password. I will rephrase it.

File: tasks/main.yml (hunk 1 above)
Comment (yakimant) [reply tasks/main.yml]: There is a setting, which has nothing to do with users/access:
```
UPDATE okm_config
SET cfg_value = 'com.openkm.workflow.adapter.JbpmRemoteWorkflowAdapter'
WHERE cfg_key = 'workflow.adapter'
```

So it looks more like config, but inside of DB.

**Pull request merged**